lesson6_task2.py

```diff
@@ -0,0 +1,32 @@
+"""
+Exclusive common numbers.
+
+Generate 2 lists with the length of 10 with random integers from 1 to 10, and make a third list containing the common
+integers between the 2 initial lists without any duplicates.
+Constraints: use only while loop and random module to generate numbers
+"""
+from random import randint
+
+my_list1 = []
+my_list2 = []
+my_list3 = []
+
+length = 0
+
+while length < 10:
```

Comment:
Here you need to slightly modify the behaviour of your program: you only check for equality of two simultaneously generated numbers, but there could be duplicates on different indexes.

You should consider using approach like:
1. Populate two lists
2. To get difference between two lists you can convert them to sets and get a difference between two sets, like:
```python 
set(my_list1) ^ set(my_list2)  # XOR operator to get only those present in one of the sets
```